app/models/tag.rb

```diff
@@ -10,14 +10,24 @@ class Tag < ApplicationRecord
   validates_presence_of :user, :name, :type
   validates :name, uniqueness: { scope: :type }
 
+  # TODO
+  # Label can apply to Posts
+  # Setting can apply to Posts and Characters with descriptions
+  # Warnings can apply to Posts
+  # Canon can apply to Posts and Settings with descriptions, original/non, creator
+  # GalleryGroups can apply to Galleries
+  # CharacterGroups can apply to Characters < should combine into Group?
```

Comment:
some user-specific tag which applies to either galleries or characters sounds good (i.e. a Group, but distinct from a regular tag since I made GalleryGroups by grafting bits and pieces onto tags)